git.py

```diff
@@ -20,15 +23,26 @@ def create_pr(branch_full_name,
         "base": base_branch
     })
 
-    url = GIT_API_URL.format(owner=owner, project=project)
+    url = GIT_API_CREATE_PR_URL.format(owner=owner, project=project)
     header = {'Authorization': 'token {api_token}'.format(api_token=api_token)}
 
-    response = requests.post(url, data=data, headers=header)
-
-    if response.status_code in [200, 201]:
-        return response.json().get('html_url')
-
-    raise Exception(response.json().get('message'))
+    try:
+        response = requests.post(url, data=data, headers=header)
+        response.raise_for_status()
+    except HTTPError as e:
+        if e.response is None:
+            raise
+        message = e.response.json().get('message')
+        raise Exception(str(e) + '. Message: ' + message)
+
+    response_data = response.json()
+    html_url = response_data.get('html_url')
+    pr_number = response_data.get('number')
+    if isinstance(labels, list) and len(labels) > 0:
```

Comment:
can we have this as a separated function like `add_labels`.
and probably have to be called from main not as part of create pr